packet/arp.go

```diff
@@ -41,18 +43,18 @@ func (hdr *ARPHdr) String() string {
     PLen:  %d\n
     Operation: %d\n
     Sender MAC address: %02x:%02x:%02x:%02x:%02x:%02x\n
-    Sender IPv4 address: %d.%d.%d.%d\n
```

Comment:
Changing IPv4 format from decimal to hex is wrong. Traditionally IPv4 addresses are printed in decimal.